packages/ui-extensions/src/surfaces/customer-account/components/Menu/Menu.doc.ts

```diff
@@ -0,0 +1,74 @@
+import type {ReferenceEntityTemplateSchema} from '@shopify/generate-docs';
+
+const data: ReferenceEntityTemplateSchema = {
+  name: 'Menu',
+  description:
+    'Use a menu to display a list of actions in a popover. Actions can open a modal, trigger an event, or link to an external page.',
+  thumbnail: 'menu-thumbnail.png',
+  requires: '',
+  isVisualComponent: true,
+  type: '',
+  definitions: [
+    {
+      title: 'MenuProps',
+      description: '',
+      type: 'MenuPropsDocs',
+    },
+    {
+      title: 'ButtonProps children',
```

Comment:
I dont think we restrict the props anymore for Button

Reply:
We do override some props https://github.com/shop/world/blob/main/areas/clients/checkout-web/packages/checkout-web-ui/src/components/Button/withContextualStyle.tsx#L22-L27. So even if they pass some styling, it won't be reflected.